packages/app-mobile/root.tsx

```
@@ -295,7 +295,9 @@ class AppComponent extends React.Component<AppComponentProps, AppComponentState>
 	private unsubscribeScreenWidthChangeHandler_: EmitterSubscription|undefined;
 	private unsubscribeNetInfoHandler_: NetInfoSubscription|undefined;
 	private unsubscribeNewShareListener_: UnsubscribeShareListener|undefined;
-	private onAppStateChange_: ()=> void;
+	private onAppStateChange_: (nextAppState: AppStateStatus)=> void;
```

ESLint

[error] 'nextAppState' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)